esmvaltool/cmorizers/data/downloaders/datasets/woa.py

```diff
@@ -0,0 +1,43 @@
+"""Script to download WOA from its webpage."""
+import logging
+
+from esmvaltool.cmorizers.data.downloaders.wget import WGetDownloader
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=invalid-name
+def download_dataset(config, dataset, _, __, overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    downloader = WGetDownloader(
+        config=config,
+        dataset=dataset,
+        overwrite=overwrite,
+    )
+    downloader.tier = 2
```

Comment:
Why is the `tier` hardcoded in several download scripts but not all? In fact, this is the case only for Tier2 datasets.

I'm asking because I have used `esmvaltool data download DATASET` for all Tier2 datasets for which an automatic download procedure is offered in this PR. Except a few minor comments, everything worked fine and that's great 👍  The only thing is that quite a few datasets landed in my `path_to_RAWOBS/Tier3` directory instead of `path_to_RAWOBS/Tier2`. The faulty datasets are: `CowtanWay`, `CRU`, `Duveiller2018`,  `E-OBS`,  `Eppley-VGPM-MODIS`,  `PATMOS-x`,  `PERSIANN-CDR`,  `REGEN`,  `Scripps-CO2-KUM`. In all these cases, `downloader.tier = 2` is not part of the script. Could you please check that?

Reply:
This is now passed to the downloader as readed from the datasets.yml